src/style/global-style.js

```diff
@@ -34,6 +34,33 @@ button {
   50% { opacity: 0.6 }
   to { opacity: 0.85 }
 }
+
+a {
+  border-left: 1px solid #dddddd;
+}
+
+*:focus {
+  /* Provide a fallback style for browsers
+    that don't support :focus-visible */
+  outline: 2px solid red;
+  background: transparent;
+}
+
+@supports selector(*:focus-visible) {
+  *:focus, button:focus, select:focus, input:focus, a:focus, svg:focus  {
+    /* Remove the focus indicator on mouse-focus for browsers
+       that do support :focus-visible */
+    outline: none !important;
+    background: transparent;
+  }
+}
+
+*:focus-visible, button:focus-visible, select:focus-visible, input:focus-visible, a:focus-visible, svg:focus-visible {
+  border: 5px solid darkorange;
+  border-radius: 0.25rem;
+  box-sizing: border-box;
```

Comment:
Unfortunately, this border thing moves content around. Using `outline` is the typical solution as the outline never takes up any space in any calculation or layout. So adding an outline never moves content around. You can also place the outline inside the element with `outline-offset: <negative-value>`. However, outline can't have a radius. In that case, you could use `box-shadow: 0 0 0 5px inset darkorange`. This only clashes if applied to elements that already have a box-shadow. But not sure if that's the case here.

But I would say that this content moving around while tabbing should be fixed :/